homepage/_packages/cloud2edge/tour.md

```diff
@@ -63,27 +66,27 @@ The demo device's digital twin supports a temperature property which will be set
 by means of the following command:
 
 {% clipboard %}
-curl -i -u demo-device@org.eclipse.packages.c2e:demo-secret -H 'application/json' --data-binary '{
+curl -i -k -u demo-device@org.eclipse.packages.c2e:demo-secret -H 'application/json' --data-binary '{
```

Comment:
FYI: I've added the `-k` (i.e. `--insecure`) option here for now. Better would be to fetch the used CA cert beforehand (via `kubectl get secret`) and use that here instead. But it seems the CA cert is currently only set in a K8s secret if Kafka messaging is being used. That means a generic solution requires changes also to the Hono chart.